apps/src/accounts/AccountInformation/style.module.scss

```diff
@@ -0,0 +1,41 @@
+@import '@cdo/apps/componentLibrary/common/styles/mixins';
+@import '@cdo/apps/componentLibrary/typography/typography.module';
+
+.inputContainer {
+  display: flex;
+  flex-direction: column;
+  max-width: 400px;
+  gap: 1.125rem;
+  margin: 1rem 0;
+
+  label.input {
+    width: 100%;
+
+    &.emailInput {
+      margin-bottom: 0.5rem;
+    }
+
+    &.authorizedTeacher {
+      font-weight: bold;
+      color: $light_affirmative_500;
```

Comment:
The "verified teacher" check box and text is showing up black for me now instead of green like it used to.

Staging:
<img width="160" alt="Screenshot 2024-10-09 at 12 09 23 PM" src="https://github.com/user-attachments/assets/0343d799-0681-4a8d-9749-6a68b48e530b">

This branch:
<img width="153" alt="Screenshot 2024-10-09 at 12 12 45 PM" src="https://github.com/user-attachments/assets/710cd395-07db-42e5-963e-d245d802194f">

If this was intentional feel free to ignore this comment.

Reply:
hmm yeah that doesn't seem right... thanks for finding this!

Reply:
much better! thanks again!

![Screenshot 2024-10-09 at 1 55 43 PM](https://github.com/user-attachments/assets/b7963fc8-be9e-4be7-ae6f-316080880985)